odl_video/settings.py

```diff
@@ -104,6 +104,9 @@ def get_var(name, default):
 # Application definition
 
 INSTALLED_APPS = [
+    'ui.apps.UIConfig',
+    'cloudsync.apps.CloudSyncConfig',
+    'dj_elastictranscoder',
```

Comment:
I have no preferences, but I was wondering if you looked for possible ones.
I remember from reading documentation for unrelated issues that the sequence is pretty important, so I thought you looked into it.
Anyway I think we will find out.